crates/delta/src/routes/servers/member_edit.rs

```diff
@@ -159,5 +212,43 @@ pub async fn edit(
         )
         .await?;
 
+    if let Some(new_voice_channel) = new_voice_channel {
+        if let Some(channel) = get_user_voice_channel_in_server(&target_user.id, &server.id).await? {
+            let old_node = get_channel_node(&channel).await?.unwrap();
+
+            let new_node = match get_channel_node(new_voice_channel.id()).await? {
+                Some(node) => node,
+                None => {
+                    set_channel_node(new_voice_channel.id(), &old_node).await?;
+                    old_node.clone()
+                }
+            };
+
+            set_user_moved_from_voice(&channel, new_voice_channel.id(), &target_user.id).await?;
+
+            let mut query = perms(db, &target_user).channel(&new_voice_channel);
+            let permissions = calculate_channel_permissions(&mut query).await;
+
+            voice_client.create_room(&new_node, &new_voice_channel).await?;
+            let token = voice_client.create_token(&new_node, &target_user, permissions, &new_voice_channel)?;
+
+            voice_client.remove_user(&old_node, &target_user.id, &channel).await?;
+
+            EventV1::UserMoveVoiceChannel {
+                node: new_node,
+                token
+            }
+            .p_user(target_user.id.clone(), db)
+            .await;
```

Comment:
Will this cause problems if online clients receive it and are not connected to voice? Or will clients just have to ignore this if not connected.